shell/platform/darwin/common/framework/Headers/FlutterHourFormat.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_FRAMEWORK_HEADERS_FLUTTERHOURFORMAT_H_
+#define FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_FRAMEWORK_HEADERS_FLUTTERHOURFORMAT_H_
+
+#import <Foundation/Foundation.h>
+
+@interface FlutterHourFormat : NSObject
```

Comment:
Why is this a class? Classes that have nothing but static members is generally an anti-pattern, and unlike Java, Objective-C has stand-alone functions.

Reply:
Discussed over chat but I think it's reasonable for this to be a class in order to ease testing (injecting a fake in the mac test).